requirements/requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
 -r requirements.txt
 sounddevice
 numpy
-sphn
+sphn==0.1.12
```

Comment:
major breaking changes in `0.2.0`...not sure why. in the future might be worth moving to a different (better supported) package for audio use cases

Reply:
Ah yeah iirc this was primarily to handle the PCM file format. Probably something better out there!